public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
- [ ] Please try to remove the boilerplate files. They unnecessarily add up to your production bundle.